### PR TITLE
Show Performance Overlay Debug Control

### PR DESCRIPTION
**Pull request by natefrechette**
This PR adds the performance overlay option to the debug controls list. When turned on, the monarch preview window will display flutters performance overlay.
Screenshot 2023-08-31 at 7 24 49 PM

### CODE REVIEW

**Comment by fertrig** [comment]
I published monarch_definitions version 1.4.0. You can use that version in this PR. 

I think you will also need another PR for the change to monarch_grpc. Let me know when you have new changes. Thanks!

**Comment by fertrig** [comment]
I published monarch_grpc version 2.3.0.

File: controller/pubspec.yaml
Comment (fertrig) [inline controller/pubspec.yaml:1]: I'm not sure why this is showing in the diff. This change is already in main branch.

File: packages/monarch_grpc/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: monarch_grpc
 description: gRPC interfaces for Monarch. Monarch is a tool for building Flutter widgets in isolation. It makes it easy to build, test and debug complex UIs.
-version: 2.2.0
+version: 2.3.0
```
Comment (fertrig) [inline packages/monarch_grpc/pubspec.yaml:3]: I don't see this new version referenced from other monarch modules, like the controller or preview_api.

File: preview_api/lib/src/preview_api_service.dart
Comment (fertrig) [inline preview_api/lib/src/preview_api_service.dart:1]: This file is failing the `dart format` analysis.

**Review by fertrig — CHANGES_REQUESTED** [review]

File: preview_api/lib/src/preview_api_service.dart
Comment (natefrechette) [reply preview_api/lib/src/preview_api_service.dart:1]: Done

File: packages/monarch_grpc/pubspec.yaml (hunk 1 above)
Comment (natefrechette) [reply packages/monarch_grpc/pubspec.yaml]: done, forgot to update

**Review by fertrig — APPROVED** [review]

**Pull request merged**